packs/feats/marshall-fiendish-forces.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>You have been in contact with a retinue of fiends, brokering all manner of deals so that they may eventually serve you. When you Manifest your Realm, you can choose to have one such fiend come to your aid. This fiend is a common fiend of level 11 or lower, which you select when you take this feat. The fiend appears in a place of your choosing within your manifested realm and has the minion trait. They remain for as long as your realm is manifested, until they're reduced to 0 Hit Points, or until you command them to leave, at which point they immediately vanish and return to their place of origin.</p>\n<p>At 20th level, you can choose a common fiend of level 15 or lower to appear instead when you Manifest your Realm.</p>"
+            "value": "<p>You have been in contact with a retinue of fiends, brokering all manner of deals so that they may eventually serve you. When you @UUID[Compendium.pf2e.actionspf2e.Item.Manifest Realm]{Manifest your Realm}, you can choose to have one such fiend come to your aid. This fiend is a common fiend of level 11 or lower, which you select when you take this feat. The fiend appears in a place of your choosing within your manifested realm and has the minion trait. They remain for as long as your realm is manifested, until they're reduced to 0 Hit Points, or until you command them to leave, at which point they immediately vanish and return to their place of origin.</p>\n<p>At 20th level, you can choose a common fiend of level 15 or lower to appear instead when you Manifest your Realm.</p>"
```

Comment:
I wonder if this is intended to be named "Marshal Fiendish Forces"? That's how it's named in another place.